modules/ROOT/pages/data-modeling.adoc

```diff
@@ -127,25 +120,6 @@ Its main purpose is to enable the replicator to determine which data to sync and
 
 Each document change is assigned a unique revision ID. The IDs of past revisions are available. The content of past revisions may be available if the revision was created locally and the database has not yet been compacted.
 
-[[Scopes-and-Collections]]
-== Scopes and Collections
```

Comment:
I'm not quite sure why you removed this, but I think you mentioned PM was in agreement?
If so, then 👍

Reply:
Yeah I got some feedback that this was not in the right place. So it's all been shifted to 'whats new' 'Confg' and 'Sync'